packages/components/src/list/style.scss

```diff
@@ -53,7 +54,7 @@
 	$background-color: $white;
 	$background-color-hover: $light-gray-100;
 	$border-color: $light-gray-tertiary;
-	$foreground-color: $theme-color;
+	$foreground-color: var(--wp-admin-theme-color);
```

Comment:
use the new css var

Reply:
`$theme-color` was derived from `theme(button)` in Gutenberg and evaluated in using postCSS. In our codebase we have variations like `theme(primary)` or `theme(outlines)`. Those may need to be replaced as well.